src/test/miner_tests.cpp

```diff
@@ -81,7 +81,7 @@ CBlockIndex CreateBlockIndex(int nHeight)
 
 bool TestSequenceLocks(const CTransaction &tx, int flags)
 {
-    LOCK(mempool.cs);
+    LOCK2(cs_main, mempool.cs);
```

Comment:
The lock is already acquired in src/test/miner_tests.cpp:208, no?

Reply:
Good point! Fixed!